src/main/resources/assets/inventorytweaks/ItemTree.xml

```diff
@@ -118,13 +122,18 @@
                 </stone>
                 <cobblestone id="cobblestone"/>
                 <mossStone id="mossy_cobblestone"/>
-                <endStone id="end_stone"/>
-                <whiteStone id="end_stone"/>
+				<endStones>
+					<endStone id="end_stone"/>
+					<endStoneBricks id="end_bricks"/>
```

Comment:
I'm not sure these count as 'natural' -- note the absence of stone bricks (even though they do generate in strongholds)